### PR TITLE
Move away from the experimental headers

### PR DESCRIPTION
**Pull request by pleroy**
First step towards addressing #1777.

### CODE REVIEW

**Comment by pleroy** [comment]
retest this please

**Comment by pleroy** [comment]
retest this please

**Comment by pleroy** [comment]
retest this please

File: install_deps.sh [hunk 1]
```diff
@@ -71,19 +71,10 @@ git checkout master
 git pull
 popd
 
-if [ ! -d "eggsperimental_filesystem" ]; then
-  git clone "https://github.com/Norgg/eggsperimental_filesystem.git"
+if [ ! -d "compatilibity" ]; then
```
Comment (eggrobin) [inline install_deps.sh]: Typo.

**Pull request merged**